examples/agents_tracing-eval_mlflow/nps_agent/langgraph_nps.py

```
@@ -0,0 +1,78 @@
+# Cell 2: Initialize MCP Client and get tools
+client = MultiServerMCPClient(
+    {
+        "nps": {
+            "transport": "sse",
+            "url": NPS_MCP_URL,
+        }
+    }
+)
+
+# Get tools from MCP server
+tools = await client.get_tools()
```

Ruff (0.15.0)

[error] `await` statement outside of a function

(F704)